app/forms/instance-create.tsx

```diff
@@ -669,6 +747,131 @@ const AdvancedAccordion = ({
+            </TipIcon>
+          </h2>
+          {isFloatingIpAttached && (
+            <MiniTable.Table>
+              <MiniTable.Header>
+                <MiniTable.HeadCell>Name</MiniTable.HeadCell>
+                <MiniTable.HeadCell>IP</MiniTable.HeadCell>
+                {/* For remove button */}
+                <MiniTable.HeadCell className="w-12" />
+              </MiniTable.Header>
+              <MiniTable.Body>
+                {attachedFloatingIpsData.map((item, index) => (
+                  <MiniTable.Row
+                    tabIndex={0}
+                    aria-rowindex={index + 1}
+                    aria-label={`Name: ${item.name}, IP: ${item.ip}`}
+                    key={item.name}
+                  >
+                    <MiniTable.Cell>{item.name}</MiniTable.Cell>
+                    <MiniTable.Cell>{item.ip}</MiniTable.Cell>
+                    <MiniTable.Cell>
+                      <button
+                        onClick={() => {
+                          detachFloatingIp(item.name)
+                        }}
+                      >
+                        <Error16Icon title={`remove ${item.name}`} />
+                      </button>
+                    </MiniTable.Cell>
+                  </MiniTable.Row>
+                ))}
+              </MiniTable.Body>
+            </MiniTable.Table>
+          )}
+          <div>
+            <Button size="sm" className="shrink-0" onClick={addFloatingIpPlaceholder}>
+              Attach floating IP
+            </Button>
+          </div>
+
+          {!!attachedFloatingIps.length && (
+            <Modal
+              isOpen={isEmptyFloatingIpPlaceholderPresent}
+              onDismiss={removeFloatingIpPlaceholder}
+              title="Attach floating IP"
+            >
+              <Modal.Body>
+                <Modal.Section>
+                  {availableFloatingIps.length > 0 && (
+                    <Message
+                      variant="info"
+                      content={
+                        <>
+                          {'This instance will be reachable at '}
+                          {selectedFloatingIp ? (
+                            <HL>
+                              {
+                                attachableFloatingIps.find(
+                                  (ip) => ip.name === selectedFloatingIp
+                                )?.ip
+                              }
+                            </HL>
+                          ) : (
+                            'the selected IP'
+                          )}
+                        </>
+                      }
+                    />
+                  )}
+                  <form>
+                    <Listbox
+                      name="floatingIp"
+                      items={availableFloatingIps.map((i) => ({
+                        value: i.name,
+                        label: (
+                          <>
```

Comment:
fragment unnecessary because there's only one div in here. I guess if it works with just the fragment, might as well do that